src/store_rebuild.cc

```diff
@@ -244,9 +244,10 @@ void
 Progress::print(std::ostream &os) const
 {
     if (goal > 0) {
+        const auto savedPrecision = os.precision(2);
         const auto percent = 100.0 * completed / goal;
-        os << std::setprecision(2) << percent << "% (" <<
-            completed << " out of " << goal << ")";
+        os << percent << "% (" << completed << " out of " << goal << ")";
+        os.precision(savedPrecision);
```

Comment:
```suggestion
       (void)os.precision(savedPrecision);
```
since we ignore the return value. However, I am not sure that we maintain this rule in the code.

Reply:
We do not enforce it. But it is good practice, so if you want to please do.